README.rst

```diff
@@ -31,42 +31,61 @@
 
 |
 
-PyScaffold helps you setup a new Python project. Checkout out `this demo project`_, which was set up using Pyscaffold.
-In order to install PyScaffold, just pick your favourite installation method::
+PyScaffold is the tool of choice for bootstrapping high quality Python
+packages, ready to be shared on PyPI_ and installed via pip_.
+It is easy to use and encourages the adoption of the best tools and
+practices of the Python ecosystem, helping you (and your team, if your have one)
+to stay sane, happy and productive. The best part? It is stable and been used
+in the wild for over half a decade!
 
-    # Good old pip
+Checkout out `this demo project`_, which was set up using PyScaffold.
+If you are not convinced yet, have a look on these `reasons to use PyScaffold`_
+
+Quickstart
+==========
+
+Just pick your favourite installation method::
+
+    ## Good old pip
+    # (make sure it is up-to-date: pip install -U pip setuptools)
     pip install pyscaffold
 
-    # Conda for the datascience fans
+    ## Conda for the datascience fans
     conda install -c conda-forge pyscaffold
 
-    # Or even pipx for the virtualenv aficionados
+    ## Or even pipx for the virtualenv aficionados
     pipx install pyscaffold
 
 If you want to install all PyScaffold's *extensions* you can even::
 
-        pip install pyscaffold[all]
+    pip install pyscaffold[all]
 
 (More details of each method are available in the `installation docs`_)
 
-This will give you a new ``putup`` command and you can just type::
+After the installation, a new ``putup`` command will be available and you can just type::
 
     putup my_project
 
 This will create a new folder called ``my_project`` containing a perfect *project
-template* with everything you need for some serious coding. After the usual::
+template* with everything you need for some serious coding.
+After ``cd``-ing into your new project you can do the usual `editable install`_::
 
-    pip install -U pip setuptools setuptools_scm
```

Comment:
Installing PyScaffold in a brand new virtualenv works, without the need for using setuptools_scm, so I suppose a sufficiently updated version of pip/setuptools will read the minimal setup requiremens from pyproject and use them in editable mode.